tools/install.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+read -p 'Enable ssl? Press Enter for none: ' ssl
```

Comment:
We're specifically enabling `letsencrypt` so question should reflect that.
We also need an email if that's set to true (alternatively - just ask for email and enable letsencrypt if anything is passed)